modules/sirdataRtdProvider.js

```diff
@@ -370,57 +369,24 @@ export function addSegmentData(reqBids, data, moduleConfig, onDone) {
           case 'msq_classic':
           case 'msq_max':
           case '366_apx':
```

Comment:
I think I'm missing some context, if the intent of this is to remove bidder-specific handling why is there a giant switch statement for a bunch of bidders?